src/EnergyPlus/ChillerAbsorption.cc

```diff
@@ -651,7 +651,7 @@ void BLASTAbsorberSpecs::setupOutputVars(EnergyPlusData &state)
                                 OutputProcessor::SOVStoreType::Summed,
                                 this->Name,
                                 {},
-                                "DistrictHeatingSteam",
+                                fluidNameSteam,
```

Comment:
@dareumnam If I'm following correctly,I think this should be "DistrictHeatingSteam". This for for SetupOutputVariable is only used it the chiller is not connected to a plant loop, so it is consuming "DistrictHeatingSteam" directly. If the chiller is conncted to a plant loop, then the SetupOutputVariable call above is used which sets the resource type to PLANTLOOPHEATINGDEMAND.